admin/app/controllers/spree/admin/gift_cards_controller.rb

```
@@ -0,0 +1,91 @@
+        @search = @search.redeemed if params[:q][:status_eq] == 'redeemed'
+
+        @search = @search.where(code: params[:q][:code_eq].strip) if params[:q][:code_eq].present?
+
+        @search = @search.ransack(params[:q])
+
+        @collection = @search.result
+
+        @collection = @collection.page(params[:page]).per(params[:per_page]) if params[:format] != 'csv'
+
+        @collection
+      end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for collection is too high. [<12, 47, 13> 50.22/45]

(Metrics/AbcSize)